app/addons/documents/base.js

```diff
@@ -42,14 +42,18 @@ function getQueryParam (query) {
   return query;
 }
 
+function partitionUrlComponent(partitionKey) {
+  return partitionKey ? '/_partition/' + partitionKey : '';
```

Comment:
Can you rather do `return partitionKey ? "/_partition/${partitionKey}" : '';`
I've had to add in the wrong " at the top because of markdown.